Yoast/wordpress-seo
# Add image fallbacks

## Summary


This PR can be summarized in the following changelog entry:

* When term has an image in its content use that one as fallback.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Add some content with an image for the term the description.
* See that image being rendered as image meta for Twitter and for Facebook

## UI changes

## Documentation

## Quality assurance


Fixes #
